### PR TITLE
OCI examples remove reactive API

### LINKED ISSUE #6859: Update OCI examples

### PR DESCRIPTION
**Pull request by aserkes**
Fixes #6859 

Replacement for PR https://github.com/helidon-io/helidon/pull/6944

### CODE REVIEW

File: examples/integrations/oci/atp/src/main/java/io/helidon/examples/integrations/oci/atp/OciAtpMain.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    public static void main(String[] args) throws IOException {
+        // load logging configuration
+        LogConfig.configureRuntime();
+
+        // By default this will pick up application.yaml from the classpath
+        config = Config.create();
+
+        WebServer server = WebServer.builder()
+                .routing(OciAtpMain::routing)
+                .port(config.get("server.port").asInt().orElse(8080))
```
Comment (romain-grecourt) [inline examples/integrations/oci/atp/src/main/java/io/helidon/examples/integrations/oci/atp/OciAtpMain.java:54]: Replace with `.config(config.get("server"))`

File: examples/integrations/oci/vault/src/main/java/io/helidon/examples/integrations/oci/vault/VaultService.java [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+import com.oracle.bmc.keymanagement.KmsCrypto;
+import com.oracle.bmc.keymanagement.responses.DecryptResponse;
+import com.oracle.bmc.keymanagement.responses.EncryptResponse;
+import com.oracle.bmc.keymanagement.responses.SignResponse;
+import com.oracle.bmc.keymanagement.responses.VerifyResponse;
+import com.oracle.bmc.secrets.Secrets;
+import com.oracle.bmc.secrets.responses.GetSecretBundleResponse;
+import com.oracle.bmc.vault.Vaults;
+import com.oracle.bmc.vault.responses.CreateSecretResponse;
+import io.helidon.common.Base64Value;
```
Comment (romain-grecourt) [inline examples/integrations/oci/vault/src/main/java/io/helidon/examples/integrations/oci/vault/VaultService.java:35]: import ordering is wrong. Groups are `java`, `helidon`, `others`

File: examples/integrations/oci/vault/src/main/java/io/helidon/examples/integrations/oci/vault/OciVaultMain.java [hunk 3]
```diff
@@ -14,25 +14,24 @@
...
 
+import com.oracle.bmc.keymanagement.KmsCrypto;
+import com.oracle.bmc.keymanagement.KmsCryptoClient;
+import com.oracle.bmc.secrets.Secrets;
+import com.oracle.bmc.secrets.SecretsClient;
+import com.oracle.bmc.vault.Vaults;
+import com.oracle.bmc.vault.VaultsClient;
 import io.helidon.logging.common.LogConfig;
 import io.helidon.config.Config;
-import io.helidon.reactive.webserver.Routing;
```
Comment (romain-grecourt) [inline examples/integrations/oci/vault/src/main/java/io/helidon/examples/integrations/oci/vault/OciVaultMain.java:23]: imports ordering is wrong

File: examples/integrations/oci/objectstorage/src/main/java/io/helidon/examples/integrations/oci/objecstorage/ObjectStorageService.java [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import com.oracle.bmc.ConfigFileReader;
+import com.oracle.bmc.auth.AuthenticationDetailsProvider;
+import com.oracle.bmc.auth.ConfigFileAuthenticationDetailsProvider;
+import com.oracle.bmc.objectstorage.ObjectStorageClient;
+import io.helidon.common.http.Http;
```
Comment (romain-grecourt) [inline examples/integrations/oci/objectstorage/src/main/java/io/helidon/examples/integrations/oci/objecstorage/ObjectStorageService.java:31]: import ordering is wrong

File: examples/integrations/oci/metrics/src/main/java/io/helidon/examples/integrations/oci/telemetry/OciMetricsMain.java [hunk 5]
```diff
@@ -14,32 +14,30 @@
...
 import java.time.temporal.ChronoUnit;
 import java.util.Arrays;
 import java.util.Date;
 import java.util.HashMap;
 import java.util.Map;
-import java.util.concurrent.CountDownLatch;
 
+import com.oracle.bmc.monitoring.Monitoring;
+import com.oracle.bmc.monitoring.MonitoringClient;
 import io.helidon.logging.common.LogConfig;
```
Comment (romain-grecourt) [inline examples/integrations/oci/metrics/src/main/java/io/helidon/examples/integrations/oci/telemetry/OciMetricsMain.java:28]: import ordering is wrong

File: examples/integrations/oci/atp/src/main/java/io/helidon/examples/integrations/oci/atp/AtpService.java [hunk 6]
```diff
@@ -31,17 +32,21 @@
...
 
+import com.oracle.bmc.ConfigFileReader;
+import com.oracle.bmc.auth.AuthenticationDetailsProvider;
+import com.oracle.bmc.auth.ConfigFileAuthenticationDetailsProvider;
+import com.oracle.bmc.database.Database;
+import com.oracle.bmc.database.DatabaseClient;
 import io.helidon.common.http.Http;
-import io.helidon.common.reactive.Single;
 import io.helidon.config.Config;
+import io.helidon.config.ConfigException;
```
Comment (romain-grecourt) [inline examples/integrations/oci/atp/src/main/java/io/helidon/examples/integrations/oci/atp/AtpService.java:42]: import ordering is wrong

File: examples/integrations/oci/objectstorage/src/main/java/io/helidon/examples/integrations/oci/objecstorage/OciObjectStorageMain.java [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+     */
+    public static void main(String[] args) {
+
+        config = Config
+                .builder()
+                .sources(examplesConfig())
+                .build();
+
+        WebServer server = WebServer.builder()
+                .routing(OciObjectStorageMain::routing)
```
Comment (romain-grecourt) [inline examples/integrations/oci/objectstorage/src/main/java/io/helidon/examples/integrations/oci/objecstorage/OciObjectStorageMain.java:58]: `.config(config.get("server"))`

File: examples/integrations/oci/vault/src/main/java/io/helidon/examples/integrations/oci/vault/OciVaultMain.java [hunk 8]
```diff
@@ -70,28 +69,28 @@ public static void main(String[] args) throws IOException {
...
+                        .register("/vault", new VaultService(secrets,
+                                vaults,
+                                crypto,
+                                vaultOcid,
+                                compartmentOcid,
+                                encryptionKey,
+                                signatureKey))
+                        .error(BmcException.class, (req, res, ex) -> res.status(
+                                ex.getStatusCode()).send(ex.getMessage())))
+                .port(config.get("server.port").asInt().orElse(8080))
```
Comment (romain-grecourt) [inline examples/integrations/oci/vault/src/main/java/io/helidon/examples/integrations/oci/vault/OciVaultMain.java:89]: Replace with `.config(config.get("server"))`

File: examples/integrations/oci/atp/src/main/java/io/helidon/examples/integrations/oci/atp/AtpService.java (hunk 6 above)
Comment (aserkes) [reply examples/integrations/oci/atp/src/main/java/io/helidon/examples/integrations/oci/atp/AtpService.java:42]: How do you find it? :)

**Review by romain-grecourt — APPROVED** [review]

**Pull request merged**